sshd-common/src/main/java/org/apache/sshd/common/config/keys/loader/openssh/OpenSSHKeyPairResourceParser.java

```diff
@@ -18,11 +18,14 @@
  */
 package org.apache.sshd.common.config.keys.loader.openssh;
 
+import static java.text.MessageFormat.format;
```

Comment:
We do not allow static imports - the code does not seem to be compiled (which would have made `checkstyle` complain).

Reply:
The code _is_ compiled (and run -- of course I tested that the new tests work). But I don't see any checkstyle warnings in Eclipse. Probably would need to install some checkstyle plugin...